config/prod.exs

```diff
@@ -11,12 +11,21 @@ use Mix.Config
 # before starting your production server.
 config :become, BecomeWeb.Endpoint,
   http: [:inet6, port: System.get_env("PORT") || 4000],
-  url: [host: "example.com", port: 80],
+  url: [scheme: "https", host: "limitless-lake-66848", port: 443],
+  secret_key_base: System.get_env("SECRET_KEY_BASE"),
+  force_ssl: [rewrite_on: [:x_forwarded_proto]],
   cache_static_manifest: "priv/static/cache_manifest.json"
 
 # Do not print debug messages in production
 config :logger, level: :info
 
+
```

Comment:
The app requires the creation of a database table that isn't used. 
We should have generated the context with --no-schema, or removed the 
unused migration after generating.